.github/workflows/milestone-analysis.yml

```diff
@@ -12,19 +12,14 @@ jobs:
   milestone:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
-      - uses: actions/setup-python@v5
-        with:
-          python-version: "3.11"
-          cache: "pip"
-      - name: Install dependencies
+      - name: Check milestone
         run: |
-          python -m pip install --upgrade pip
-          pip install -r requirements.txt
-          pip install -r tasks/libs/requirements-github.txt
-      - name: Run PR check for milestone assignment
-        env:
-          BRANCH_NAME: ${{ github.ref_name }}
-          PR_ID: ${{ github.event.pull_request.number }}
-        run: |
-          inv -e lint-milestone
+          milestones="${{join(github.event.pull_request.milestone.*.name, ' ')}}"
```

Comment:
Can there be more than a single milestone?

If not, I guess there's no need for the loop, and if it's possible, then the current code will accept the triage milestone if it's set with another one